scripts/coverage.js

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env node
```

Comment:
this entire script is basically just `yarn lerna run coverage`, right?

Reply:
It appears so! Thanks - again wasn't aware that was possible so good to know